pkgs/by-name/pi/pimsync/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromSourcehut,
+  pkg-config,
+  sqlite,
+  scdoc,
+  makeWrapper,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pimsync";
+  version = "0.1.0";
+
+  src = fetchFromSourcehut {
+    owner = "~whynothugo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-upOCrpbveSSFrhdHDkTOmja4MLmsgtuoDHMsgXyulWI=";
+  };
+
+  cargoHash = "sha256-QRgyHzIjdI8+OzvM+3sfuxTNMVCdGxrMvQpCnot9iUM=";
+
+  nativeBuildInputs = [
+    pkg-config
+    scdoc
+    makeWrapper
+  ];
+
+  buildInputs = [
+    sqlite
+  ];
+
+  buildPhase = ''
+    make build
+  '';
+
+  installPhase = ''
+    make install PREFIX=$out
+  '';
```

Generally, you want to avoid editing the phase themselves:
```suggestion
  makeFlags = [
    "build"
    "PREFIX=${placeholder "out"}"
  ];
```